src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lendable\Polyfill\Symfony\MessengerBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Bundle\FullStack;
+use Symfony\Component\Serializer\Serializer;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('lendable_polyfill_messanger')
+            ->fixXmlConfig('transport')
+            ->fixXmlConfig('bus', 'buses');
+
+        $rootNode->children()
+            ->arrayNode('routing')
+            ->useAttributeAsKey('message_class')
+            ->beforeNormalization()
+            ->always()
+            ->then(static function ($config) {
```

Comment:
```suggestion
            ->then(static function ($config): array {
```
I appreciate that this code wasn't crafted by you.